File: four_pillars_of_oop/no_01_encapsulation/no_05_flexibility/python/Temperature.py
class Temperature:
    def __init__(self, celsius):
        self.__celsius = celsius  # Private attribute

    def to_fahrenheit(self):
        return (self.__celsius * 9/5) + 32

    def set_celsius(self, celsius):
        self.__celsius = celsius

    def get_celsius(self):
        return self.__celsius

File: four_pillars_of_oop/no_01_encapsulation/no_05_flexibility/java/Temperature.java
package four_pillars_of_oop.no_01_encapsulation.no_05_flexibility.java;

public class Temperature {
    private double celsius;  // Private attribute

    public Temperature(double celsius) {
        this.celsius = celsius;
    }

    public void setCelsius(double celsius) {
        this.celsius = celsius;
    }

    public double getCelsius() {
        return celsius;
    }

    public double toFahrenheit() {
        return (celsius * 9/5) + 32;
    }
}
